### PR TITLE
Fix: update Organization business rules

### PR DESCRIPTION
**Pull request by juliusfitzhugh-ccpo**
Business rules pointing to an undefined record for DISA agency in SNOW sandbox.
Updated with the current record for DISA in sandbox
- `Org - Only DISA has a 'DISA Org.'`
- `Org - Org. Name null if Agency is DISA`

Ticket: AT-8359

### CODE REVIEW

File: f600233d1b154d507b782f84604bcb12/update/sys_script_796665eb871281108ea6cbb9cebb35f8.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
         <condition/>
         <description/>
         <execute_function>false</execute_function>
-        <filter_condition table="x_g_dis_atat_organization">agency=19ba0874db9a5950b1227ea5f39619bf^organization_nameISNOTEMPTY^EQ<item display_table="x_g_dis_atat_agency" display_value="Defense Information Systems Agency (DISA)" endquery="false" field="agency" goto="false" newquery="false" operator="=" or="false" value="19ba0874db9a5950b1227ea5f39619bf"/>
+        <filter_condition table="x_g_dis_atat_organization">agency=fdd5245e875a9910bc86b889cebb35e3^organization_nameISNOTEMPTY^EQ<item display_table="x_g_dis_atat_agency" display_value="Defense Information Systems Agency (DISA)" endquery="false" field="agency" goto="false" newquery="false" operator="=" or="false" value="fdd5245e875a9910bc86b889cebb35e3"/>
```
Comment (juliusfitzhugh-ccpo) [inline f600233d1b154d507b782f84604bcb12/update/sys_script_796665eb871281108ea6cbb9cebb35f8.xml:18]: Simply updated referenced record in the BR.

File: f600233d1b154d507b782f84604bcb12/update/sys_script_7cd27203871e01108ea6cbb9cebb35a8.xml [hunk 2]
```diff
@@ -15,7 +15,7 @@
         <condition/>
         <description/>
         <execute_function>false</execute_function>
-        <filter_condition table="x_g_dis_atat_organization">agency!=19ba0874db9a5950b1227ea5f39619bf^disa_organization!=^EQ<item display_table="x_g_dis_atat_agency" display_value="Defense Information Systems Agency (DISA)" endquery="false" field="agency" goto="false" newquery="false" operator="!=" or="false" value="19ba0874db9a5950b1227ea5f39619bf"/>
+        <filter_condition table="x_g_dis_atat_organization">agency!=fdd5245e875a9910bc86b889cebb35e3^disa_organization!=^EQ<item display_table="x_g_dis_atat_agency" display_value="Defense Information Systems Agency (DISA)" endquery="false" field="agency" goto="false" newquery="false" operator="!=" or="false" value="fdd5245e875a9910bc86b889cebb35e3"/>
```
Comment (juliusfitzhugh-ccpo) [inline f600233d1b154d507b782f84604bcb12/update/sys_script_7cd27203871e01108ea6cbb9cebb35a8.xml:18]: Simply updated referenced record in the BR.

File: f600233d1b154d507b782f84604bcb12/update/sys_script_796665eb871281108ea6cbb9cebb35f8.xml (hunk 1 above)
Comment (juliusfitzhugh-ccpo) [reply f600233d1b154d507b782f84604bcb12/update/sys_script_796665eb871281108ea6cbb9cebb35f8.xml:18]: Example 
OrganizationBR-bug_AT-8359

**Review by nkingsley-ccpo — APPROVED** [review]

**Comment by jasonburkert-ccpo** [comment]
New agency value for DISA checks out
image

**Pull request merged**